src/app/cube/content-pages/contribute-page/contribute-page.component.html

```diff
@@ -0,0 +1,30 @@
+<div>
+  <div class="header">
+      <h1>
+          {{ title }}
+      </h1>
+  </div>
+<div class = 'accordion'>
```

Comment:
```suggestion
<div class="accordion">
```

Reply:
Done and did the same thing to `<mat-tab-group class = 'tabs'` on line 8.